### ISSUE TITLE
small UX improvement (right panel)

### ISSUE THREAD

**Issue by DJCrashdummy** [vector-im/riot-web#8688]
beforehand: congrats for the redesign! :clap: - after a few minutes adjustment to it, it looks & feels much cleaner and straight forward. :+1: 

but one thing seems pretty insane to me from the UX-point of view: why isn't in the right panal the invite-button swapped with the filter?!?

- filters are usually placed above lists they filter (like in the left panel for the rooms)
- this would make this UI even more consistent (both filters are above its list and even at the same hight)
- i'm pretty sure the filter is more often used (considering all users and time - only the admin/mod invites a user just once, but many user look for him probably more than once)
- the only argument about not swapping these two may be, that newbies probably won't find the invite-button if it is "hidden" at the right bottom... BUT as this is the only coloured button, this won't happen often. - if you're looking for it, you'll find it. :wink:

**Comment by lampholder** [comment]
Thanks for the suggestion 😁 

I think you've intuited correctly why the invite button is where it is (I think the bottom right is the least looked-at part of any UX if I'm remembering correctly).

We'll certainly bear this in mind as we iterate on the UX!

**Comment by DJCrashdummy** [comment]
well, there is only one argument (against many others) to not swap these two buttons... and as long as the invite-button is the only coloured button in the whole UI it is even a very weak one!

**Comment by nuflowx** [comment]
On the note of the invite-button, it's very attention grabbing as it's the only coloured thing of the entire interface, so for me it's rather distracting.

Just my first impression as a new user

**Comment by DJCrashdummy** [comment]
yes @nuflowx, this also crossed my mind... another reason why i personally would move it to the bottom.

**Comment by yusf** [comment]
Came here to support switching filter and invite button.

**Comment by yannicka** [comment]
I also think that the filter should be at the top and the invitation button at the bottom.

**Comment by DJCrashdummy** [comment]
@yannicka instead of (unnecessarily) pinging all participants, you can simply :+1: the initial post... this may even help to prioritize this issue because they can be sorted by reactions to the initial post. (and if you want to get pinged in the future, simply use the *subscribe* button on the right side.)
